test/e2e/replacement_values.py

```diff
@@ -63,6 +72,13 @@
     "eu-west-3": "447278800020.dkr.ecr.eu-west-3.amazonaws.com",
     "me-south-1": "986000313247.dkr.ecr.me-south-1.amazonaws.com",
     "sa-east-1": "818342061345.dkr.ecr.sa-east-1.amazonaws.com",
+    "eu-south-1": "563282790590.dkr.ecr.eu-south-1.amazonaws.com",
+    "ap-northeast-3": "479947661362.dkr.ecr.ap-northeast-3.amazonaws.com",
+    "af-south-1": "314341159256.dkr.ecr.af-south-1.amazonaws.com",
+    "eu-south-2": "",
```

Comment:
when does the key error occur? it means we cannot run a particular test in this region

shouldnt the test fail to find the image instead of this map returning an empty value?

Reply:
Removed placeholder values